experimental/dds/tree/.eslintrc.js

```diff
@@ -0,0 +1,65 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+module.exports = {
+	extends: ['@fluidframework/eslint-config-fluid/eslint7'],
+	// This setting is the same as in fluid. However, specifying it here causes the parser module to be evaluated relative
+	// to shared-tree, which references @typescript-eslint/parser@~4.2.0 rather than @2.17.0. The former supports more advanced array
+	// types which are used in several modules.
+	parser: '@typescript-eslint/parser',
+	root: true,
+	rules: {
+		// Recover "noUnusedLocals" behavior as part of linting.
+		'@typescript-eslint/no-unused-vars': ['error', { args: 'none', varsIgnorePattern: '^_' }],
+
+		'@typescript-eslint/quotes': [
+			'error',
+			'single',
+			{
+				allowTemplateLiterals: true,
+				avoidEscape: true,
+			},
+		],
+		// Intentionally not unifying signatures can enable more scoped API documentation and a better developer experience, which accounts
+		// for all violations of this rule in this package at the time of writing.
+		'@typescript-eslint/unified-signatures': 'off',
+
+		// Prettier
+		'comma-dangle': 'off',
+		'max-len': 'off',
+
+		// Rules which could be re-enabled (by dropping these overrides, as they are enabled in base config) with some minor fixes:
+		'@typescript-eslint/strict-boolean-expressions': 'off',
+		'no-shadow': 'off',
+		'prefer-arrow/prefer-arrow-functions': 'off',
+		'no-null/no-null': 'off', // Payloads use null
+		'no-redeclare': 'off', // Persisted type factories need to be classes to pass the typescript version of this rule
+	},
```

Comment:
(I'll volunteer to look at these as well.)

Reply:
These rules are all just style differences in the package. Do you feel strongly about enabling any of them?

Reply:
I'd prefer all the stylistic rules be removed and use the defaults in the shared config.

Reply:
We're working toward that, but I'd prefer to make those changes in a future PR. We'll want them in the internal repository as well (and I'd like to minimize the set of symmetrical changes to typescript made in this initial commit).

Reply:
No worries. Given the temporary constraints we're working under this is a fine compromise.